python/tests/graph/test_utils.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import tensorflow as tf
+
+import sparkdl.graph.utils as tfx
+
+from ..tests import PythonUnitTestCase
+
+class TFeXtensionGraphUtilsTest(PythonUnitTestCase):
+
```

Comment:
each test function should test a particular aspect of a particular function in the tfx module, so it's easy to understand what exactly is being tested, and thus if a test fails it's immediately obvious what exactly failed.  could you restructure so that is true? e.g. a few I see embedded in the existing tests below are:
```
def test_op_name():
  # test that op_name from an op name equals input

def test_op_name_from_tensor():
  # test that op_name from a tensor is correct

...
```
If there is shared set-up code, you can make helper functions for them.

Reply:
pending on the result on test refactoring in PR-1